### DISCUSSION TITLE
vscode extension copilot runtime status shows not yet activated

### DISCUSSION THREAD

**Discussion by 421518** [community/community#31383] [Copilot Conversations] [answered]
I paid $10 on August 22nd and about a month ago the copilot icon in the bottom right corner of my vscode stopped showing. The extention copilot runtime status shows not yet activated. I want to know what is the reason, please contact email 421518@qq.com. Many thanks

**Comment by 421518** [comment]
And the extension icon does not appear at all in the bottom left corner of the vscode ide。But everything is fine on my laptop and my remote server。My PC is using VPN, but it still doesn't work after I turn VPN off.

**Comment by davecheney** [comment, marked as answer]
@421518 i'm sorry you're having difficulty. 
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by 421518** [reply]
![image](https://user-images.githubusercontent.com/102877451/187588232-45f48fc4-83ca-49b1-9e9d-d5a395dedb63.png)
The icon does not appear after installing Copilot. View -> Output Copilot option does not have any output

**Reply by davecheney** [reply]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic? If so, please consult https://github.com/orgs/community/discussions/29127

**Reply by 421518** [reply]
It solved my problem perfectly!!!!!!https://github.com/orgs/community/discussions/29127
